components_library/cachehierarchies/classic/no_cache.py

```diff
@@ -113,6 +113,8 @@ def incorporate_cache(self, board: AbstractBoard) -> None:
                 int_req_port = self.membus.mem_side_ports
                 int_resp_port = self.membus.cpu_side_ports
                 core.connect_interrupt(int_req_port, int_resp_port)
+            else:
+                core.connect_interrupt()
```

Comment:
Just to clarify, this holds true for all non-X86 targets?

Reply:
It's definitely true that all targets must have `BaseCPU.createInterrupts()` called on it. Whether this is the right place to do it or not is up for debate :)